### PR TITLE
Backport node_name fixes to 2023.2.4

### LINKED ISSUE #544: UI: backport node_name fixes into 2023.2
Backport solution applied to Issue #492 into 2023.2

### PR DESCRIPTION
**Pull request by HeriLFIU**
Closes #544 

### Summary

Backported node_name fixes to `base/2023.2.4`. If the switch names are available, they will show instead of the dpids.

### Local Tests

The mef_eline UI was tested and no errors occurred.

### CODE REVIEW

**Comment by HeriLFIU** [comment]
@viniarck

**Comment by Alopalao** [comment]
Nice PR, I tested it and seems to be working as expected. Two things though:
- There is already a version `2023.2.4` with a small change but necessary.
- Lets also apply this [PR](https://github.com/kytos-ng/mef_eline/pull/486) it is for the list of EVCs.

**Review by Alopalao — CHANGES_REQUESTED** [review]
Nice PR, I tested it and seems to be working as expected. Two things though:
- There is already a version `2023.2.4` with a small change but necessary.
- Lets also apply this [PR](https://github.com/kytos-ng/mef_eline/pull/486) it is for the list of EVCs.

**Comment by viniarck** [comment]
> Nice PR, I tested it and seems to be working as expected. Two things though:
> 
> * There is already a version `2023.2.4` with a small change but necessary.
> * Lets also apply this [PR](https://github.com/kytos-ng/mef_eline/pull/486) it is for the list of EVCs.

Good observations and great review. 

Regarding `2023.2.4`, can you elaborate on what you mean? It [hasn't been bumped yet](https://github.com/kytos-ng/mef_eline/releases/tag/2023.2.4) or any mentioning regarding any other in-flight conflicting one? 

Also, `kytos.json` needs be bumped, Heriberto.

**Comment by Alopalao** [comment]
I did not realize it was not released. I made a mistake with the branch [2023.2.4](https://github.com/kytos-ng/mef_eline/tree/base/2023.2.4). The fix is already applied to 2023.2.3.

**Comment by HeriLFIU** [comment]
@Alopalao oh, so it was already backported?

**Review by viniarck — COMMENTED** [review]
Looks good to me. I noticed you mentioned the local tests, did you also have the chance to try it out on 2023.2 build too, Heriberto, just to confirm?

I'll let @Alopalao to finish the review, it's great to also have his review. The backport looks good, the other fixes were done in two PRs one fixing `show_circuit.kytos` [on PR 494](https://github.com/kytos-ng/mef_eline/pull/494/) and `list_connections.kytos` [on PR 486](https://github.com/kytos-ng/mef_eline/pull/486), initially on version `2024.1` during its development it was only fixed for one component, but in your backport you addressed both from what I followed based on Aldo's suggestion, let's wait for Aldo confirmation too @HeriLFIU.

**Comment by Alopalao** [comment]
Sorry for the confusion, the [PR 494](https://github.com/kytos-ng/mef_eline/pull/494) and [PR 486](https://github.com/kytos-ng/mef_eline/pull/486) are not backported to 2023. My confusion was that I saw a branch with 2023.2.4 and I assumed the version was already released, it is not. Proceed this PR with the version 2023.2.4 and please do apply [PR 486](https://github.com/kytos-ng/mef_eline/pull/486) here as well. Both issues have very similar approaches.

**Comment by HeriLFIU** [comment]
@viniarck I hadn't tried out the new changes on 2023.2, Imma test them now and finish the pr.

**Comment by HeriLFIU** [comment]
@Alopalao Just tested out all of the UI elements and the changes themselves and everything seems to be in line. I got no errors and the switch names are displayed.
image

**Review by Alopalao — APPROVED** [review]
Great adittion, tested the changes and the dpid names are being waited. I will approve this PR. Do not forget to change the source branch back to 2023.2.3 so it merges with the last release.

**Review by viniarck — APPROVED** [review]
img

Great PR, and great collaboration, guys. Let's ship it.

**Pull request merged**
